lib/castle_devise/patches/registrations_controller.rb

```diff
@@ -8,8 +8,60 @@ module Patches
     module RegistrationsController
       extend ActiveSupport::Concern
 
-      included do
-        before_action :castle_filter, only: :create
+      # @param klass [self]
+      def self.prepended(klass)
+        klass.class_eval do
+          before_action :castle_filter, only: :create
+        end
+      end
+
+      # PUT /resource
+      def update
+        context = CastleDevise::Context.from_rack_env(request.env, scope_name, resource)
```

Comment:
Isn't `resource` `nil` here?

Reply:
Could you add a test to make sure that `context` has a proper `resource` when sending requests to Castle?